qcodes/instrument_drivers/Lakeshore/Model_336.py

```diff
@@ -1,14 +1,26 @@
+from enum import Enum
+
 from qcodes import VisaInstrument, InstrumentChannel, ChannelList
-from qcodes.utils.validators import Enum, Strings
+from qcodes.utils.validators import Enum as QCEnum, Strings
+
+class ChannelDescriptor(Enum):
```

Comment:
I may be missing something important, but this seems like an unnecessary complication to add. The cons are: reduced code readability (renaming `Enum` to `QCEnum`, having to look up `ChannelDescriptor`).

What are the pros?

In any case, the naming conflict is easily resolved by either just importing `enum` or importing `qcodes.utils.validators` as `vals` (or both). `QCEnum` is an abonimation.

Reply:
Agreed, `QCEnum` is a really bad name - the state of the code is really WIP. 
But how is it a complication? I agree that it could also just be a dictionary. But does this make it simpler?